src/HelixServer.js

```diff
@@ -84,6 +84,7 @@ function executeTemplate(ctx) {
     selector: ctx._selector,
     extension: ctx._extension,
     rootPath: ctx._mount,
+    body: ctx._body,
```

Comment:
it should populate all properties in `ctx._body`, eg:

```
const params = Object.assign({ .. }, ctx._body);
```

Reply:
instead of passing the body, I will add all top level items in the request body to the context.